src/tekton/addtrigger.ts

```diff
@@ -83,10 +87,14 @@ export async function k8sCreate(trigger: TriggerTemplateKind | EventListenerKind
   }
   const fsPath = path.join(tempPath, `${trigger.metadata.name}.yaml`);
   await fs.writeFile(fsPath, triggerYaml, 'utf8');
-  const result = await TektonItem.tkn.execute(Command.create(`${quote}${fsPath}${quote}`));
-  if (result.error) vscode.window.showErrorMessage(`Fail to deploy Resources: ${getStderrString(result.error)}`);
+  const result = await cli.execute(Command.create(`${quote}${fsPath}${quote}`));
+  if (result.error) {
+    vscode.window.showErrorMessage(`Fail to deploy Resources: ${getStderrString(result.error)}`);
+    return null;
```

Comment:
I think it would be more correct syntactically if you set all returns to booleans. I also saw there is a `return;` above this one.

Reply:
Thanks, Luca I will do the changes :)